Source/PeleLMeX_Plot.cpp

```diff
@@ -951,50 +952,58 @@ PeleLM::initLevelDataFromPlt(int a_lev, const std::string& a_dataPltFile)
 
   ProbParm const* lprobparm = prob_parm_d;
 
-  // Enforce rho and rhoH consistent with temperature and mixture
-  // The above handles species mapping (to some extent), but nothing enforce
-  // sum of Ys = 1 -> use N2 in the following if N2 is present
+  // If m_do_patch_flow_variables is set as true, call user-defined function to
+  // patch flow variables
+  if (m_do_patch_flow_variables) {
+    patchFlowVariables(geom[a_lev], *lprobparm, ldata_p->state);
+  } else {
```

Comment:
Should this be en `else`? I feel like the correction below should happen even when patchFlowVariables is being used. That would allow the user to be "lazy" and for exmaple just redefine temperature and count on the logic below to appropriately reset enthalpy.

Reply:
yes, you are right. it is better to keep it that way.
change incorporated